test/ops/array_ops_test.py

```diff
@@ -672,6 +683,9 @@ def testTensorSliceEagerMemory(self):
   @test_util.assert_no_new_pyobjects_executing_eagerly
   @test_util.assert_no_garbage_created
   def testVariableSliceEagerMemory(self):
+    if sys.version_info.major == 3 and sys.version_info.minor == 11:
+      # TODO(b/265082239)
```

Comment:
Generally we don't want to keep the TF TODOs since it pollutes the list of our own TODOs